json-guides/microprofile-config.json

```diff
@@ -48,7 +48,7 @@
                 "Injecting a configuration value this way is recommended for mandatory properties. If no configured value exists for this property, a <code>DeploymentException</code> will be thrown during startup."
             ],
             "instruction": [
-                "To inject a <code>download-url</code> configuration property into the code, add the following <code>@Inject</code> and <code>@ConfigProperty</code> annotations to line 6 before declaring the <code>String download-url</code>, or click  <action title='Inject a ConfigProperty'><b>@Inject @ConfigProperty(name=\"download-url\")</b></action>. Then, click <action title='Save'><b>Save</b></action> on the editor menu pane.<br><br><note>Note: You have not yet specified a value for the <code>download-url</code> configuration property.  Execution of this code at this time would cause a <code>DeploymentException</code> to be thrown since a value is required.</note>"
+                "To inject a <code>download-url</code> configuration property into the code, add the following <code>@Inject</code> and <code>@ConfigProperty</code> annotations to line 10 before declaring the <code>String download-url</code>, or click  <action title='Inject a ConfigProperty' onclick='microprofileConfigCallBack.addInjectConfigButton(event)'><b>@Inject @ConfigProperty(name=\"download-url\")</b></action>. Then, click <action title='Run' onclick=\"microprofileConfigCallBack.saveButton(event)\"><b>Run</b></action> on the editor menu pane.<br><br><note>Note: You have not yet specified a value for the <code>download-url</code> configuration property.  Execution of this code at this time would cause a <code>DeploymentException</code> to be thrown since a value is required.</note>"
```

Comment:
Change download-url to download_url for the property name. Change "String download-url" to "private String downloadUrl".